help.go

```diff
@@ -29,6 +29,10 @@ func (e *Executor) tasksWithDesc() (tasks []*taskfile.Task) {
 	tasks = make([]*taskfile.Task, 0, len(e.Taskfile.Tasks))
 	for _, task := range e.Taskfile.Tasks {
 		if task.Desc != "" {
+			compiledTask, err := e.CompiledTask(taskfile.Call{Task: task.Task, Vars: task.Vars})
```

Comment:
Passing `Vars` here is not needed. This is for call vars; task vars are used automatically.

```suggestion
			compiledTask, err := e.CompiledTask(taskfile.Call{Task: task.Task})
```